course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -90,11 +90,103 @@ describe('Unit testing PeopleController', function() {
         it('won\'t bother waiting for profile lookups or POSTing new members ' +
             'if course membership lookup fails');
     });
+
     describe('addNewMember', function() {
-        it('won\'t add person if they already have a course enrollment');
-        it('won\'t add if they could not be found via /people lookup');
-        it('won\'t add if multiple profiles returned by /people lookup');
-        it('adds person if not in course and single profile available');
+
+        beforeEach(function() {
+            controller = $controller('PeopleController', {$scope: scope });
+            this.members = {};
+            this.members["12345678"] = [{
+                        active: 1,
+                        email_address: "test45@mcelroy.org",
+                        name_first: "Test",
+                        name_last: "User",
+                        prime_role_indicator: "Y",
+                        role_type_cd: "STUDENT",
+                        univ_id: "12345678"
+                    }];
+            this.members["43215678"] = [{
+                        active: 1,
+                        email_address: "test46@mcelroy.org",
+                        name_first: "Peter",
+                        name_last: "Smith",
+                        prime_role_indicator: "Y",
+                        role_type_cd: "STUDENT",
+                        univ_id: "43215678"
+                    }];
+
+            this.personResult = [
+                {
+                    active: 1,
+                    email_address: "test45@mcelroy.org",
+                    name_first: "Test",
+                    name_last: "User",
+                    prime_role_indicator: "Y",
+                    role_type_cd: "STUDENT",
+                    univ_id: "12345678"
+                },
+                ["test45@mcelroy.org"]
+            ];
+        });
+        afterEach(function() {
+            // handle the course instance get from setTitle, so we can always
```

Comment:
There's now a [one-liner](https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/blob/task/elliottyates/tlt-2119/course_info_add_multiple_people_jasmine_tests/course_info/tests/jasmine/controllers/PeopleControllerSpec.js#L39) for this:
```
clearInitialCourseInstanceFetch();
```